### ISSUE TITLE
Translation 

### ISSUE THREAD

**Issue by kawazzzzzz** [mudlej/mj_pdf#19]
Hi, 

Is there any way to participate in the translation of this great application ?
I can translate into french if you want, let me know if you need help. 🙂

Kawaz.

**Comment by mijorus** [comment]
Same

**Comment by mudlej** [comment]
Yes, see recent MRs on GitLab such as [this](https://gitlab.com/mudlej_android/mj_pdf_reader/-/merge_requests/18).
That said, a french translation just got [merged](https://gitlab.com/mudlej_android/mj_pdf_reader/-/merge_requests/18).
